test/geocel/PolygonUtils.test.cc

```diff
@@ -0,0 +1,119 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020-2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file geocel/detail/PolygonUtils.test.cc
+//---------------------------------------------------------------------------//
+#include "geocel/detail/PolygonUtils.hh"
+
+#include "celeritas_test.hh"
+
+namespace celeritas
+{
+namespace test
+{
+//---------------------------------------------------------------------------//
+
+using Real2 = Array<real_type, 2>;
+using VecReal2 = std::vector<Real2>;
+using Real3 = Array<real_type, 3>;
+using celeritas::axpy;
+using namespace celeritas::geoutils;
```

Comment:
Instead of `using namespace` like this, change the test namespace so that you're always in the `test` namespace underneath whatever functions you're testing. So for a file testing functions defined in `celeritas::orangeinp`, the test namespace should be `celeritas::orangeinp::test`.